tests/helpers.js

```diff
@@ -134,7 +134,8 @@ export function makeSureItFails(): Function {
  * assert.equal(fakeProcess.exit.called, true);
  *
  */
-export function fake(original: Object, methods: Object = {}): Object {
+ // $FLOW_IGNORE: fake can return any kind of object and fake a defined set of methods for testing.
+export function fake<T>(original: Object, methods: Object = {}): T {
```

Comment:
The above change seems to be needed to convince flow that this `fake` test helper can return any kind of objects
(it helps to be able to use the tricks used in the testing suite without loosing the additional type checking in the actual code)